app/src/main/res/values/strings.xml

```diff
@@ -232,6 +232,10 @@ limitations under the License.
     <string name="pref_key_myexperiment_pass">pref_myexperimentpass</string>
     <string name="pref_summary_myexperiment_pass">Myexperiment password</string>
 
+    <!--Disclaimer:-->
+    <string name="DisclaimerTitle">Disclaimer:</string>
+    <string name="DisclaimerContent">myExperiment's Logo is nothing to do with Apache and is copyright to myExperiment</string>
```

Comment:
same as above strings resource name should be `disclaimer_content` not `DisclaimerContent`.